.travis.yml

```
@@ -1,8 +1,9 @@
 sudo: false
 rvm:
-  - 2.0.0
-  - 2.1.8
   - 2.2.2
   - 2.3.0
+  - 2.4.0
+  - 2.5.0
+  - 2.6.0
+  - 2.7.0
```

```suggestion
  - "2.5"
  - "2.6"
  - "2.7"
```

A nice way to allow rvm to keep the patch versions up-to-date is to rely on its "aliases", example:

https://github.com/rvm/rvm/blob/master/config/known#L12

---

This Suggestion would match this Savon change, https://github.com/savonrb/savon/commit/b644b513bb7255681631c5ed39a8781404c35135